### ISSUE TITLE
Search not work

### ISSUE THREAD

**Issue by vingocheng** [microsoft/vscode#200522]
Type: <b>Bug</b>

I found vscode searching not work any more.
when I try to search, it pop a red background hint like 'The "path" argument must be of type string. Received type undefined'
I not change anything, and it can work well before.

VS Code version: Code 1.85.0 (af28b32d7e553898b2a91af498b1fb666fdebe0c, 2023-12-06T20:48:09.019Z)
OS version: Windows_NT x64 10.0.19045
Modes:
Remote OS version: Linux x64 4.4.0-19041-Microsoft

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz (8 x 1690)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.73GB (3.99GB free)|
|Process Argv|--crash-reporter-id 70b34912-2fcf-41db-b845-0014d243fc33|
|Screen Reader|no|
|VM|67%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu-16.04|
|OS|Linux x64 4.4.0-19041-Microsoft|
|CPUs|11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz (8 x 1690)|
|Memory (System)|15.73GB (3.99GB free)|
|VM|67%|
</details><details><summary>Extensions (33)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|13.4.2
numbered-bookmarks|ale|8.4.0
jupyter-keymap|ms-|1.1.2
remote-containers|ms-|0.327.0
remote-ssh|ms-|0.107.1
remote-ssh-edit|ms-|0.86.0
remote-wsl|ms-|0.81.8
vscode-remote-extensionpack|ms-|0.25.0
remote-explorer|ms-|0.4.1
remote-server|ms-|1.5.0
gitlens|eam|2023.12.904
copilot|Git|1.141.0
copilot-chat|Git|0.11.0
vscode-pull-request-github|Git|0.76.1
python|ms-|2023.22.0
vscode-pylance|ms-|2023.11.10
cmake-tools|ms-|1.16.32
cpptools|ms-|1.18.5
cpptools-extension-pack|ms-|1.3.0
hexeditor|ms-|1.9.12
makefile-tools|ms-|0.7.0
java|red|1.25.1
cmake|twx|0.0.17
vscode-lldb|vad|1.10.0
intellicode-api-usage-examples|Vis|0.2.8
vscodeintellicode|Vis|1.2.30
vscode-java-debug|vsc|0.55.0
vscode-java-dependency|vsc|0.23.3
vscode-java-pack|vsc|0.25.15
vscode-java-test|vsc|0.40.1
vscode-maven|vsc|0.43.0
debug|web|0.26.1
vscode-proto3|zxh|0.5.5

(2 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30410667
py29gd2263:30899288
vscaat:30438848
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyone:30548225
3biah626:30602489
f6dab269:30613381
a9j8j154:30646983
showlangstatbar:30737416
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
welcomedialog:30910333
pythonnosmt12:30797651
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
dsvsc013:30795093
dsvsc014:30804076
dsvsc015:30845448
pythontestfixt:30902429
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
pythontbext0:30879054
accentitlementsc:30887149
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
aa_t_chat:30882232

```

</details>

**Comment by vingocheng** [comment]
![image](https://github.com/microsoft/vscode/assets/105616986/e9e16b66-4c70-4311-af3a-5e3043a93cc7)

**Comment by vingocheng** [comment]
![image](https://github.com/microsoft/vscode/assets/105616986/c4e7f437-5e2d-4ff3-9735-a79db4928eb7)

**Comment by jerkovicl** [comment]
Same for me, search not working with new version

**Comment by andreamah** [comment]
Does this only happen on your remote? Is this on WSL?

**Comment by andreamah** [comment]
> ![image](https://private-user-images.githubusercontent.com/105616986/289449918-c4e7f437-5e2d-4ff3-9735-a79db4928eb7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzMzU4MzIsIm5iZiI6MTcwMjMzNTUzMiwicGF0aCI6Ii8xMDU2MTY5ODYvMjg5NDQ5OTE4LWM0ZTdmNDM3LTVlMmQtNGZmMy05NzM1LWE3OWRiNDkyOGViNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxMVQyMjU4NTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kNzAzNWU0OWIwOTg3NDU4MmM1YmY3ZmI4NDgyMjQ0MmUxMzE3NGQ1N2Y5ZmJiMGFlNjM0ZDAxOWM2NDE0NTZiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.41w7c0yrndikmX0QduC3k_DrGnj-2J1TMJNckO9e464)

@jerkovicl are you getting the message in this screenshot?

**Comment by vingocheng** [comment]
> Does this only happen on your remote? Is this on WSL?

Issue is there on local FS.

**Comment by vingocheng** [comment]
> Does this only happen on your remote? Is this on WSL?

In my case, it only happened on WSL. (but it can work well before)

**Comment by vingocheng** [comment]
I remove WSL extension from vocode and the issue is gone.
So, it may relate with the extension.

**Comment by jerkovicl** [comment]
@andreamah nope, i am not getting that error, i dont see any red overlays around search bar

**Comment by andreamah** [comment]
> I remove WSL extension from vocode and the issue is gone. So, it may relate with the extension.

Were you using WSL when this happened?

**Comment by jerkovicl** [comment]
@andreamah 

> @jerkovicl does this happen with all extensions disabled?

no it does not, so it must be some extension that is blocking search.
First suspect is this one:
![image](https://github.com/microsoft/vscode/assets/5444153/1aa1319c-43d2-45d2-b9cc-ae2e7351dcf3)

But even after disabling that extension, it didnt work and no red border  around search bar that would give clue about next extension that is not working,

**Comment by andreamah** [comment]
Do you have `search.experimental.closedNotebookRichContentResults` enabled?

**Comment by jerkovicl** [comment]
@andreamah i have `"search.experimental.notebookSearch": true,` but i see that setting is unknown configuration setting and i have `"search.experimental.closedNotebookRichContentResults": true`

**Comment by andreamah** [comment]
This was actually a bug that I just fixed here https://github.com/microsoft/vscode/issues/199606 so `search.experimental.closedNotebookRichContentResults: true` should work now.

**Comment by jerkovicl** [comment]
@andreamah awesome!

**Comment by vingocheng** [comment]
> This was actually a bug that I just fixed here #199606 so `search.experimental.closedNotebookRichContentResults: true` should work now.

Thanks for your great work
